app/helpers/sprangular/application_helper.rb

```diff
@@ -46,9 +46,9 @@ def template_paths
             .assets.each_logical_path
             .select { |file| file.end_with?('html') }
             .map do |file|
-              path = digest_assets? ? File.join('/assets', Rails.application.assets[file].digest_path) : asset_path(file)
+              path = digest_assets? ? File.join('/assets', Rails.application.assets[file].digest_path) : file
```

Comment:
Line is too long. [109/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.